indexer-app/src/cmr/indexer/data/concepts/keyword_util.clj

```diff
@@ -20,5 +20,152 @@
   [field-values]
   (->> field-values
        (mapcat prepare-keyword-field)
-       set
-       (string/join " ")))
+       (keep not-empty)
+       (apply sorted-set)
+       (string/join \space)))
+
+(defn contact-group->keywords
```

Comment:
Defines the private functions in the namespace using `defn-` to make the code more readable.

Reply:
Fixed.